jepsen/src/jepsen/checker/timeline.clj

```diff
@@ -90,6 +118,14 @@
        (reduce (fn [m p] (assoc m p (count m)))
                {})))
 
+(defn t-index
```

Comment:
We can probably just call this :index, since it's a simple integer sequence. I thought, reading this PR, that t-index meant *time* index somehow. :)

Reply:
I meant it as :timeline-index, though really it marks the subhistory... what do you think about :sub-index instead?  I use :index [here](https://github.com/jepsen-io/jepsen/pull/220/files#diff-e4e16e33aa65970d1b55ec53a96ca2a2R481) for position in the full history, and it's where I get the linkable element ids.

Reply:
We could also skip the extra state and derive subhistory index from the index of the first non-nemesis op in the subhistory, but that's maybe overcomplicating it

Reply:
Oh, I see! I was thinking you could use :index for everything, since it'll be a unique ID in any subhistory.

Reply:
Thinking abt this more, I want to go with just :sub-index. Deriving subhistory index at each op and opens up a bunch of interactions with nemeses, and because we have the whole subhistory right at the beginning and it's simple to precompute it and pass it forward.